BankAccounts.rb

```diff
@@ -3,15 +3,17 @@
 module Bank
   class Account
     attr_reader :ident, :balance, :open_date
-    attr_accessor :owner
-    def initialize(ident, balance, open_date, owner = nil)
+    def initialize(ident, balance, open_date)
       @ident = ident.to_i
       @balance = balance.to_i
-      if @balance < 0
+      @min_balance = 0
+      check_valid_balance
+      @open_date = DateTime.strptime(open_date, "%Y-%m-%d %H:%M:%S %z")
+    end
+    def check_valid_balance
```

Comment:
I like the way you used this method to check the balance, I would recommend adding whitespace between the `end` from a previous method and the `def` of a new method